pkgs/by-name/re/refine/package.nix

```
@@ -0,0 +1,90 @@
+{
+  lib,
+  fetchFromGitLab,
+  nix-update-script,
+  blueprint-compiler,
+  desktop-file-utils,
+  libadwaita,
+  meson,
+  ninja,
+  pkg-config,
+  wrapGAppsHook4,
+  python3,
+  libxml2,
+  python3Packages,
+  libportal,
+  libportal-gtk4,
+  appstream,
+  gtk4,
+  glib,
+}:
+
```

```suggestion

let
  libadwaita' = libadwaita.overrideAttrs (finalAttrs: oldAttrs: {
    version = "1.6.2-unstable-2025-01-02";

    src = oldAttrs.src.override {
      rev = "f5f0e7ce69405846a8f8bdad11cef2e2a7e99010";
      hash = "sha256-n5RbGHtt2g627T/Tg8m3PjYIl9wfYTIcrplq1pdKAXk=";
    };
  });
in
```

Looks like upstream is [using](https://gitlab.gnome.org/TheEvilSkeleton/Refine/-/blob/371932becfea4255a4fee95908d5ecf072faf15e/refine/main.py#L94-100) an [unstable](https://gnome.pages.gitlab.gnome.org/libadwaita/doc/main/method.AboutDialog.add_other_app.html) libadwaita API. We'll need to do the same